Formula/httpd.rb

```diff
@@ -0,0 +1,132 @@
+  homepage "https://httpd.apache.org/"
+  url "https://archive.apache.org/dist/httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
+    sha256 "d35e67ae745053cda23273a520ead6bde506ec8082057edcd3185c9e36eae483" => :sierra
+    sha256 "4ea7056d6c08f9cf49507869aa8c78e3522c601d87b3347d31561ba05b33d41c" => :el_capitan
+    sha256 "e6ae5c4d38b40bcf3d85877ac117a1ea048de774edec3c2b0ff714bc01a6d40a" => :yosemite
+  end
+
+  skip_clean :la
+
+  deprecated_option "with-http2" => "with-nghttp2"
+
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "apr-util"
+  depends_on "nghttp2" => :optional
+
+  if build.with?("mpm-worker") && build.with?("mpm-event")
+    raise "Cannot build with both worker and event MPMs, choose one"
+  end
+
+  def install
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    # fix non-executable files in sbin dir (for brew audit)
+    inreplace "support/Makefile.in",
+      "$(DESTDIR)$(sbindir)/envvars", "$(DESTDIR)$(sysconfdir)/envvars"
+    inreplace "support/Makefile.in",
+      "envvars-std $(DESTDIR)$(sbindir);", "envvars-std $(DESTDIR)$(sysconfdir);"
+    inreplace "support/apachectl.in",
+      "@exp_sbindir@/envvars", "#{etc}/apache2/2.4/envvars"
+
+    # install custom layout
+    (buildpath/"config.layout").unlink
+    (buildpath/"config.layout").write <<-EOS.undent
+      <Layout Homebrew>
+          prefix:        #{prefix}
+          exec_prefix:   ${prefix}
+          bindir:        ${exec_prefix}/bin
+          sbindir:       ${exec_prefix}/bin
+          libdir:        ${exec_prefix}/lib
+          libexecdir:    ${exec_prefix}/libexec
+          mandir:        #{man}
+          sysconfdir:    #{etc}/apache2/2.4
+          datadir:       #{var}/www
+          installbuilddir: ${prefix}/build
+          errordir:      ${datadir}/error
+          iconsdir:      ${datadir}/icons
+          htdocsdir:     ${datadir}/htdocs
+          manualdir:     ${datadir}/manual
+          cgidir:        #{var}/apache2/cgi-bin
+          includedir:    ${prefix}/include/httpd
+          localstatedir: #{var}/apache2
+          runtimedir:    #{var}/run/apache2
+          logfiledir:    #{var}/log/apache2
+          proxycachedir: ${localstatedir}/proxy
+      </Layout>
+    EOS
+
+    args = %W[
+      --enable-layout=Homebrew
+      --enable-mods-shared=all
+      --enable-unique-id
+      --enable-ssl
+      --enable-dav
+      --enable-cache
+      --enable-logio
+      --enable-deflate
+      --enable-cgi
+      --enable-cgid
+      --enable-suexec
+      --enable-rewrite
+      --with-apr=#{Formula["apr"].opt_prefix}
+      --with-apr-util=#{Formula["apr-util"].opt_prefix}
+      --with-pcre=#{Formula["pcre"].opt_prefix}
+      --with-ssl=#{Formula["openssl"].opt_prefix}
+      --with-z=#{Formula["zlib"].opt_prefix}
```

Comment:
We don't depend on zlib. Can we point to `/usr`?

Reply:
Will try to just remove this line.